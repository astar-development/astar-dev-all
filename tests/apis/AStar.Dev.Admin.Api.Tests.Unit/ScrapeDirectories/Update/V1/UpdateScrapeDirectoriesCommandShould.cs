using AStar.Dev.Api.Usage.Sdk.Metrics;
using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Admin.Api.ScrapeDirectories.Update.V1;

[TestSubject(typeof(UpdateScrapeDirectoriesCommand))]
public sealed class UpdateScrapeDirectoriesCommandShould
{
    [Fact]
    public void ContainTheExpectedValues()
    {
        var sut = new UpdateScrapeDirectoriesCommand();

        sut.ToJson().ShouldMatchApproved();
    }

    [Fact]
    public void ImplementIEndpointNameInterface()
    {
        var sut = new UpdateScrapeDirectoriesCommand();

        sut.ShouldBeAssignableTo<IEndpointName>();
    }
}
