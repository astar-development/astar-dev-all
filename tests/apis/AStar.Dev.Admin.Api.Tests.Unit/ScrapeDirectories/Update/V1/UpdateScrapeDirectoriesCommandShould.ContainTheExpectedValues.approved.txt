{
  "rootDirectory": "",
  "baseSaveDirectory": "",
  "baseDirectory": "",
  "baseDirectoryFamous": "",
  "subDirectoryName": ""
}